pkg/cedar/core/client_test.go

```diff
@@ -0,0 +1,65 @@
+package cedarcore
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/zap"
+	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	coremodels "github.com/cmsgov/easi-app/pkg/cedar/core/gen/models"
+)
+
+type ClientTestSuite struct {
+	suite.Suite
+	logger *zap.Logger
+}
+
+func TestClientTestSuite(t *testing.T) {
+	tests := &ClientTestSuite{
+		Suite:  suite.Suite{},
+		logger: zap.NewExample(),
+	}
+	suite.Run(t, tests)
+}
+
+func (s ClientTestSuite) TestClient() {
+	ctx := appcontext.WithLogger(context.Background(), s.logger)
+
+	ldClient, err := ld.MakeCustomClient("fake", ld.Config{Offline: true}, 0)
+	s.NoError(err)
+
+	s.Run("Instantiation successful", func() {
+		c := NewClient("fake", "fake", ldClient)
+		s.NotNil(c)
+	})
+
+	s.Run("LD defaults protects invocation", func() {
+		c := NewClient("fake", "fake", ldClient)
+		resp, err := c.GetSystemSummary(ctx)
+		s.NoError(err)
+
+		blankSummary := coremodels.SystemSummaryResponse{}
```

Comment:
`blankSummary` should be a `models.CedarSystemSummary` from our domain models, not a CEDAR model; that's what's causing the [current test failure](https://github.com/CMSgov/easi-app/runs/4431591738?check_suite_focus=true).

Reply:
Good catch! I guess I am not yet habitually running tests locally when making changes 😂  (good thing CI/CD has my back!)

Reply:
7f7ce4c897e1a8be9e62e829aca170a6a613d06f